### ISSUE TITLE
About the Le calculation in training

### ISSUE THREAD

**Issue by Zachary-ZCH** [zkawfanx/Learning-Temporal-Consistency-for-Low-Light-Video-Enhancement-from-Single-Images#10]
Thank you for your wonderful article. In the article, you pointed out that the enhancement loss Le includes two parts, the Le of (pred, target) and the Le of (pred_warped, target_warped), but in fact I only see the unwarp Le in train.py.

**Comment by zkawfanx** [comment]
Sorry, but I may not get your point by 'only see the unwarp Le in train.py'. 

In the train.py, the model takes the input then the warped input to get the 1st and 2nd outputs, then the Le of (pred, target) and Le of (pred_warped, target_warped) are calculated at L84, L91 as below.

https://github.com/zkawfanx/StableLLVE/blob/4cc070da328ffadc8498285985aa98175e473bc5/train.py#L82-L91

**Comment by Zachary-ZCH** [comment]
I see Le = ∑i=1,2‖g(xi) − yi‖1 in paper which means ：
# the 1st pass   
 pred = model(input) 
 loss = criterion(pred, target) 

Should it be expressed as follows ？

 # the 1st pass  
 pred = model(input) 
 loss = criterion(pred, target)
 input_t = warp(input, flow)
 input_t_pred = model(input_t) 
 target_t = warp(target, flow)
 loss = loss + criterion(input_t_pred , target_t )

**Comment by zkawfanx** [comment]
Yeah, you remind me of that. 

The code may miss the the supervision part for the second pass output when I clean the code up for releasing. I will check the original code sometime and feel free to try these two versions out and see what happens. 

I remember that by halving the value of lambda can also achieve similar supervision with only one part of Le, because the optical flow region is relatively small and supervising the original and warped outputs has similar effects. But the full supervision can help the model converge better and get unbiased color.

Thank you for finding out the mistake and I will fix it.
